### PR TITLE
Add files for Chrome and Firefox releases

### PR DESCRIPTION
**Pull request by spicyj**

### CODE REVIEW

File: shells/chrome/manifest.json [hunk 1]
```diff
@@ -18,8 +20,7 @@
 
   "permissions": [
     "http://*/",
-    "https://*/*",
-    "file://*"
```
Comment (kassens) [inline shells/chrome/manifest.json]: Why did you remove `file://` permission? Seems like this would allow debugging on local files that don't go through http?

Comment (spicyj) [inline shells/chrome/manifest.json]: Chrome Web Store doesn't allow it – the extension was automatically rejected. :(

File: shells/firefox/webpack.config.js [hunk 2]
```diff
@@ -9,8 +9,10 @@
  */
 'use strict';
 
+var __DEV__ = process.env.NODE_ENV !== 'production';
+
 module.exports = {
-  // devtool: 'cheap-module-eval-source-map',
+  devtool: __DEV__ ? 'cheap-module-eval-source-map' : false,
```
Comment (jaredly) [inline shells/firefox/webpack.config.js]: Firefox actually has rather poor sourcemap support generally, which is why I disabled source maps even in dev.

Comment (spicyj) [inline shells/firefox/webpack.config.js]: Oh, I missed this. I'll comment it back out.
